modules/client-configuration/pages/autoinst-advanced.xml

```diff
@@ -0,0 +1,150 @@
+[[autoinst-advanced]]
+= Advanced Techniques
+
+This section details:
+
+* how to sign metadata with your own GPG key in order to increase security on the onboarded systems, see xref:client-configuration:autoinst-advanced.adoc#include-gpg-key[Include Your Own GPG Key];
+* how to do fully unattended provisioning, see xref:client-configuration:autoinst-advanced.adoc#unattended-provisioning[Unattended Provisioning].
```

Comment:
ok

I wondered about a separate section for "advanced techniques". I did not want them to meddle in the main topic, for the sake of clarity. But I could also have done two separate sections at the end without grouping them under an "advanced techniques" roof.

Reply:
Yeah, "Advanced ..." sections are messy. What's advanced to one person might be simple to another, and it also has the unintended side effect of making people less likely to read the section (I shouldn't need it), or less likely to try the task (I can't do it). It would be better if you can be more specific with these sections, and refer to them as "optional" or "additional" rather than "advanced".